### ISSUE TITLE
Update rollup testnets

### ISSUE THREAD

**Issue by kuzdogan** [ethereum/sourcify#794]
We have support for rollup testsnets on Kovan and Rinkeby. Since these L1 testnets are deprecated the rollups moved to Goerli. Add support for Goerli rollup testnet and remove the others. Let's also add support for Gnosis (xDai) rollups without monitoring.

Optimism migrating from Kovan to Goerli: https://dev.optimism.io/kovan-to-goerli/

Arbitrum will start running on Goerli. Rinkeby deprecation planned but not until EF announces deprecation: https://www.alchemy.com/overviews/arbitrum-nitro-testnet

- [x] Add Arbitrum Goerli, with monitoring support
- [x] Remove Optimism Kovan
- [x] Remove Arbitrum Rinkeby (once Rinkeby is fully deprecated, not now)
- [x] Add Optimism on Gnosis (without monitoring)
- [x] ~~Add Arbitrum on Gnosis (without monitoring)~~

**Comment by marcocastignoli** [comment]
Arbitrum Goerli: https://github.com/ethereum/sourcify/commit/baded36535356a985cfda8199d6639555851b2ca by @kuzdogan

**Comment by marcocastignoli** [comment]
* Documentation about Optimism on Gnosis https://developers.gnosischain.com/for-developers/optimism-optimistic-rollups-on-gc
* Documentation about Abritrum on Gnosis https://developers.gnosischain.com/for-developers/developer-resources/aox-arbitrum-on-xdai

**Comment by marcocastignoli** [comment]
I believe it's still not possible to add Arbitrum on Gnosis because:
* the block explorer https://blockscout.com/xdai/aox/ (https://blockscout.com/xdai/arbitrum) **is not working**
* the rpc https://arbitrum.xdaichain.com/ **is not working**

**Comment by marcocastignoli** [comment]
Alchemy stopped supporting Ropsten, Kovan and Rinkeby
https://docs.alchemy.com/changelog/future-deprecation-of-ropsten-kovan-and-rinkeby

**Comment by marcocastignoli** [comment]
On https://testnet.arbiscan.io/ there is this message:

```📢 The Rinkeby testnet explorer has been discontinued and set to read-only on October 5th, 2022. Please migrate your contracts and deploy new ones on Goerli Arbitrum Testnet.```

**Comment by kuzdogan** [comment]
Alright, then let's remove it and close this

**Comment by marcocastignoli** [comment]
remove arbitrum rinkeby https://github.com/ethereum/sourcify/commit/31bac34d3ae1223f8533765c9362956eb209bd20
